feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/BottomSheetSessionsFragment.kt

```diff
@@ -85,6 +86,15 @@ class BottomSheetSessionsFragment : DaggerFragment() {
                 requireContext()
             )
         )
+        binding.sessionRecycler.addOnScrollListener(object : RecyclerView.OnScrollListener() {
+            override fun onScrolled(recyclerView: RecyclerView, dx: Int, dy: Int) {
+                super.onScrolled(recyclerView, dx, dy)
+
+                val scrollDirectionUp = -1
```

Comment:
Sorry for my nitpick...
Could you please define this variable as a static or put directly into the argument of #canScrollVertically?

Reply:
Fixed!